src/6_branch.js

```diff
@@ -1013,7 +1013,7 @@ Branch.prototype['sendSMS'] = wrap(
 					self._api(
 						resources.linkClick,
 						{
-							"link_url": 'l/' + url.split('/').pop(),
```

Comment:
To answer your original question more literally, the domain is stored separately and we don't need to worry about it here